<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordingSettings</class>
 <widget class="QWidget" name="RecordingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Recording</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Screen</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Select screen to capture:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="screenComboBox">
        <property name="toolTip">
         <string comment="Use this to change grabbed screen"/>
        </property>
        <property name="statusTip">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Capture Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="margin">
       <number>12</number>
      </property>
      <item>
       <widget class="QRadioButton" name="fullscreenButton">
        <property name="toolTip">
         <string comment="Grabs video from entire screen"/>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="text">
         <string>Fullscreen</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fullscreenNoAeroButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Fullscreen without Aero (Improves performance)</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="windowButton">
        <property name="toolTip">
         <string comment="Grabs Eve window"/>
        </property>
        <property name="text">
         <string>Eve window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="captureCursorCheckBox">
        <property name="text">
         <string>Capture cursor</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Quality</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Resolution:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="resolutionComboBox">
        <property name="toolTip">
         <string comment="Use this to change output video resolution"/>
        </property>
        <item>
         <property name="text">
          <string>Native</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quater Native</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1920x1080</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1280x720</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>640x480</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Encoder quality:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="qualityComboBox">
        <property name="toolTip">
         <string comment="Use this to set encoder quality"/>
        </property>
        <item>
         <property name="text">
          <string>Best</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Balanced</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Performance</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Audio grabbing</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="1">
       <widget class="QComboBox" name="primaryAudioDeviceComboBox">
        <property name="toolTip">
         <string comment="Use this to select audio device"/>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Select primary device to grab audio:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Select secondary device to grab audio:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="secondaryAudioDeviceComboBox">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
